doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -914,3 +914,4 @@ Documentation
     * :ref:`release_notes`, :ref:`software_maturity`, :ref:`known_issues`, :ref:`glossary`, and :ref:`dev-model` are now located under :ref:`releases_and_maturity`.
 
   * The :ref:`ug_thread` documentation to improve the overall presentation and add additional details where necessary.
+  * The :ref:`app_approtect` page to use the device term instead of SoC when also SiPs are referenced or mentioned.
```

Comment:
This is a very minor edit. I don't think it is relevant for the customers to know about it from the release notes.